packages/twenty-front/src/modules/object-record/record-filter/utils/isMatchingStringFilter.ts

```diff
@@ -0,0 +1,72 @@
+import { StringFilter } from '@/object-record/record-filter/types/ObjectRecordFilter';
+
+export const isMatchingStringFilter = ({
+  stringFilter,
+  value,
+}: {
+  stringFilter: StringFilter;
+  value: string;
+}) => {
+  switch (true) {
```

Comment:
not sure to see the value of switch (true) here.
if (xx) { return}
if (yy) { return } would be clearer

Reply:
I'll create a ticket to create a linting rule